DashWallet/Sources/UI/Payments/Receive/Views/ReceiveContentView.swift

```diff
@@ -20,12 +20,19 @@ import UIKit
 // MARK: - ReceiveContentView
 
 @objc(DWReceiveContentView)
-final class ReceiveContentView: UIView {
+final class ReceiveContentView: UIStackView {
```

Comment:
StackView works better here for variable content size (username can be hidden if no identity created).